subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -573,6 +575,38 @@ static int cmd_wifi_ps_timeout(const struct shell *sh, size_t argc, char *argv[]
 	return 0;
 }
 
+static int wifi_link_info(const struct shell *sh)
```

Comment:
Thinking a bit on this, these should be handled in the wifi_mgmt so that these are applicable for API as well.